QR_Code_Generator.py

```diff
@@ -0,0 +1,11 @@
+import qrcode
+import random
+
+def generate_QR(URL):
+    qr_img = qrcode.make(URL)
+    fileNum = str(random.randint(0, 999999))
+    qr_img.save("QR_" + fileNum + ".jpg")
+    print('QR Code for your link has been generated and saved successfully.')
```

Comment:
- code documentation is missing
- return the function we does not encourage using void functions